doc/base/classes.xml

```diff
@@ -257,13 +257,14 @@
 		<method name="lerp"  >
 			<return type="float">
 			</return>
-			<argument index="0" name="a" type="float">
+			<argument index="0" name="from" type="float">
 			</argument>
-			<argument index="1" name="b" type="float">
+			<argument index="1" name="to" type="float">
 			</argument>
-			<argument index="2" name="c" type="float">
+			<argument index="2" name="percent" type="float">
```

Comment:
I was struggling with what to call it, "weight" sounds good, will change it momentarily. Thanks!